README.md

```diff
@@ -1,40 +1,46 @@
 # Introduction
 
-This repository has two  examples that show how your existing Apache Spark applications can be deployed and configured seamlessly using CDAP. 
+This guide will demonstrate how to manage and run your existing Apache Spark application in CDAP. This repository has two examples and a step-by-step instructions in acheiving this. 
 
-This capability allows users of CDAP to run their existing Spark application in CDAP without making any modification to the Spark code.
+Users will be able to run their existing Spark application in CDAP without making any modifications to the Spark code.
 
-This repository contains two pre-built example spark applications.
+Sample spark examples used in this guide:
 
   - word-count-java: contains word count example written in Java 
-  - sparkpi-scala: contains sparkpi example written in Scala [how-to](SCALA-SPARK.md)
+  - sparkpi-scala: contains sparkpi example written in Scala 
 
-# How To 
+The following instructions are for deploying the word-count-java Spark application. For deploying Scala Spark app, follow the instructions [here](SCALA-SPARK.md). 
 
-This section will describe how an exisiting Apache Spark code can be integrate into CDAP. So, let us take you through a step-by-step process to get it up and running in CDAP. We will use the examples in this directory to run through it. 
+# Summary of steps
 
-## Step 1 - Building your Spark Application
+1. Package Spark application (word-count) as a bundle jar
+2. Deploy the Spark application jar as a CDAP plugin 
+3. Create a CDAP Application using the Spark plugin
+4. Run the application in CDAP   
 
-Let us assume that you have a maven project for your Apache Spark application. We have one in this repository called ```word-count-java``` that is a maven project and it's a Java Spark application. Similar to the once that you might me having in your environment. In these applications none of the dependencies of CDAP are introduced into your application, [check here](https://github.com/caskdata/sample-spark-app/blob/develop/word-count-java/src/main/java/com/example/spark/JavaWordCount.java#L1)
+## Step 1 - Package your Spark Application
 
-So, let's first build one of this project. The process will be same for any Spark based applications that you would like to run in CDAP.
+In this section the instructions are demonstrated for maven project, the steps are similar for other build tools such as ant or sbt. 
+are  that you have a maven project for your Apache Spark application. Please note, in these applications none of the dependencies of CDAP are introduced and the compiled application jar would work directly on Apache Spark: [check here](https://github.com/caskdata/sample-spark-app/blob/develop/word-count-java/src/main/java/com/example/spark/JavaWordCount.java#L1)
```

Comment:
nit: remove `are  that you have a maven project for your Apache Spark application.`.
Also the next line can be `none of the dependencies from CDAP are used`